CONTRIBUTING.md

```diff
@@ -31,19 +31,19 @@ This project uses the fork and pull workflow. For more details see [this video](
 2. Create a new branch off main when starting work on a new issue.
 3. Commit your changes in the feature branch in your local clone.
 4. Update your local repository with the most recent code from the main repository. Remember to rebase often
-5. Test your changes (if applicable) – run all existing tests and any new tests, and make sure the application works as expected. 
+5. Test your changes (if applicable) – run all existing and new tests, and make sure the application works as expected. 
 6. Create a pull request. (See below sections for a template) 
-7. Another team member must review and approve of the pull request.  
+7. Another team member must review and approve the pull request.  
 8. Merge your changes
 
 ### PR Templates
 Pull requests should be created to merge all changes. The pull request form should auto fill with the template PR file. Other team members will review the proposed code and suggest changes as required. Discussions about the code should be kept to the relevant issue/PR so that all team members can see and benefit from it.
 
 ## Code review process
-Team members in the relevant sub-team will review the code submitted for PR. Team members will review code for readability, style, bugs, test coverage and completion of acceptance criteria. The reviwers should also make ensure that the application works as expected, all prevoius tests pass, and new features have associated unit tests.
+Team members in the relevant sub-team will review the code submitted for PR. Team members will review code for readability, style, bugs, test coverage and completion of acceptance criteria. The reviewers should also ensure that the application works as expected. All previous tests pass, and new features have associated unit tests.
 
 ## Code, commit message and branch name conventions
-(When linting and formatting is finalised add details here)
+The IntelliJ IDE is reccomended for the backend to format and refactor code where necessary.
 
 ## Owners
-The different parts of the project are owned by different contributors. The owners of each sub-team should review PR's related to their area of the project. [insert links to owners files] 
+Different contributors own the different parts of the project. The owners of each sub-team should review PRs related to their area of the project.
```

Comment:
Could we please link the CODEOWNERS file here?

Reply:
Sure will do